miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditPoolController.java

```diff
@@ -128,20 +133,24 @@ public ModelAndView setupForm(@PathVariable Long poolId, ModelMap model) throws
     }
 
     if (pool == null) throw new NotFoundException("No pool found for ID " + poolId.toString());
-
+    PoolDto poolDto = Dtos.asDto(pool, true, false, errorEditDistance, warningEditDistance);
```

Comment:
The original didn't call this when it was unsaved. I'm not sure it matters either way... I'm assuming you've checked the behaviour.

Reply:
I did check it, but I checked again and found a bug in the "Pool custom" functionality so thank you :laughing: